test/unit/cache/set.test.js

```diff
@@ -0,0 +1,41 @@
+const mockClient = {
+  set: jest.fn(),
+  expire: jest.fn()
+}
+
+jest.mock('../../../app/cache/base')
+const { getClient: mockGetClient } = require('../../../app/cache/base')
+
+jest.mock('../../../app/cache/get-full-key')
+const { getFullKey: mockGetFullKey } = require('../../../app/cache/get-full-key')
+
+const { cacheConfig } = require('../../../app/config')
```

Comment:
Should this not be grouped with { set } instead? Or if not grouped, below the mocks